python/packaging/classic/setup.py

```diff
@@ -307,6 +307,7 @@ def run(self):
             "pyspark.errors",
             "pyspark.errors.exceptions",
             "pyspark.examples.src.main.python",
+            "pyspark.errors.data",
         ],
         include_package_data=True,
```

Comment:
Hmm, I assumed that since we added the new JSON file to `MANIFEST.in`, this line would pick it up for packaging. But there are finicky details related to packaging that I don't recall off the top of my head.

Reply:
There is a test at https://github.com/apache/spark/blob/master/dev/pip-sanity-check.py but that only runs a basic test that does not import `pyspark.errors` whereas https://github.com/apache/spark/actions/workflows/build_python_connect.yml tests its full compatibility (only for `pyspark-connect` package though.

Reply:
Oh that's a good point. I think we can just include that to `pyspark-connect`. Thanks for pointing this one.